client/reader/discover/style.scss

```diff
@@ -11,8 +11,4 @@
 			max-width: 968px; // Max width of dual column reader stream.
 		}
 	}
-	.discover-stream-navigation {
-		padding: 0;
-		margin-bottom: 0;
-	}
```

Comment:
This CSS applies to the header, not the DiscoverNavigation component. So, I've moved it from the component's style to a new style.scss file in `discover` root.